prow/plugins/lgtm/lgtm.go

```diff
@@ -74,18 +75,18 @@ func handle(gc githubClient, log *logrus.Entry, ic github.IssueCommentEvent) err
 	isAssignee := ic.Issue.IsAssignee(commentAuthor)
 	isAuthor := ic.Issue.IsAuthor(commentAuthor)
 	if isAuthor && wantLGTM {
-		resp := "you can't LGTM your own PR"
+		resp := "you cannot LGTM your own PR"
 		log.Infof("Commenting with \"%s\".", resp)
 		return gc.CreateComment(org, repo, number, plugins.FormatResponse(ic.Comment, resp))
 	} else if !isAuthor {
-		if !isAssignee && wantLGTM {
-			resp := "you can't LGTM a PR unless you are an assignee"
-			log.Infof("Commenting with \"%s\".", resp)
-			return gc.CreateComment(org, repo, number, plugins.FormatResponse(ic.Comment, resp))
-		} else if !isAssignee && !wantLGTM {
-			resp := "you can't remove LGTM from a PR unless you are an assignee"
-			log.Infof("Commenting with \"%s\".", resp)
-			return gc.CreateComment(org, repo, number, plugins.FormatResponse(ic.Comment, resp))
+		if !isAssignee {
+			log.Infof("Assigning %s/%s#%d to %s", org, repo, number, commentAuthor)
+			if err := gc.AssignIssue(org, repo, number, []string{commentAuthor}); err != nil {
+				resp := "Changing LGTM is restricted to assignees, and assigning you to the PR failed"
```

Comment:
When do we expect `AssignIssue` to fail? Will it fail if you're not an org member?